changelog/736.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Fix invalid widget fields.
+- :attr:`Widget.invite_url` and :attr:`Widget.fetch_invite` are now optional.
+- :attr:`WidgetMember.avatar` and :attr:`WidgetMember.activity` are no longer always ``None``.
```

Comment:
in older versions, `WidgetMember.avatar` caused an attribute error.
![image](https://user-images.githubusercontent.com/71233171/189551599-138add5f-f60d-4876-8d9f-26a2cbec9433.png)

Reply:
https://github.com/DisnakeDev/disnake/pull/736/commits/112a4dea185cc34281c35f37b87e5161092ec247